docs/best-practices.md

```diff
@@ -72,7 +72,7 @@ Apps _should_ allow all settings to be customized for each installation.
 
 ### Store configuration in the repository
 
-Any configuration _should_ be stored in the target repository. Unless the app is using files from an established convention, the configuration _should_ be stored in the `.github` directory. See the [API docs for `context.config`](https://probot.github.io/api/latest/classes/context.html#config).
+Any configuration _should_ be stored in the target repository. Unless the app is using files from an established convention, the configuration _should_ be stored in the `.github` directory. See the [API docs for `context.config`](https://probot.github.io/api/latest/modules/context.html#config).
```

Comment:
Going to switch this back to classes but fix the end to point to https://probot.github.io/api/latest/classes/context.Context.html

Reply:
611671ab17d1f6608237087c7427bd5079e00f4e

should be good now